src/compas/utilities/__init__.py

```diff
@@ -72,6 +72,7 @@
     :nosignatures:
 
     fibonacci
+    binomial_coefficient
```

Comment:
Shouldn't this be called `binomial` or the function renamed to `binomial_coefficient`? Or are these two different things?

Reply:
I think binomial_coefficient is the correct name.

Reply:
I agree, but what I meant is, you called the function `binomial` but referenced in the docs as `binomial_coefficient`, so doc will fail to find it. Unless, there's a second function called `binomial_coefficient` that I'm not seeing...

Reply:
see the warning in travis: https://travis-ci.com/compas-dev/compas/jobs/177444951#L658

Reply:
ah yes!

Reply:
I am now using binomial_coefficient everywhere